### DISCUSSION TITLE
Tuya TS0044 not showing on Dashboard

### DISCUSSION THREAD

**Discussion by dave21w** [Koenkk/zigbee2mqtt#13076] [Devices]
Just purchased a couple of these, setup was easy and the switch shows up in the devices list but never appears on the dashboard, normally at first use the new device appears then but when you press a button the dashbaord flickers briefly but that's it, all the switches have shown up in Domoticz ok and function as expected, anyone else notice this at all and if so is there a fix?

Thanks
Dave

**Comment by broekd** [comment]
Dave did you ever found the solution for this. I have the same problem here with a couple of Tuya buttons (ers-10tzbvk-aa)

**Comment by janumix** [comment]
I've got the same problem. 2 devices battery powered (TS0041 and TS0043) that work like Remote Switch also not being shown on dashboard.

Regards
Janusz

**Comment by dave21w** [comment]
Hi broekd, sorry I thought I'd replied to your question, no i never managed to resolve it.

I also have another issue which has now come to light however this maybe more to do with the wireless switch than z2m, the TS0044 can be left for a few days without being pressed and as soon as a button is used the system detects it and any script associated is run correctly, however I also have some TS0042 wireless switches which apart from also not showing on the dashboard seem to go to sleep, if they are not pressed for a number of hours the first press does nothing and you then have to press it again to get a response, even when they do work first time theres seems to be a longer delay in operation then the TS044.

**Reply by broekd** [reply]
Np when so time I will dig into it and let you know why they don't appear on the dashboard

**Comment by poto4ever** [comment]
Any news?
